courses/free-courses-en.md

```diff
@@ -330,6 +330,7 @@
 * [CryptoZombies.io](https://cryptozombies.io) - CleverFlare, Loom Network
 * [Ethereum Developer Bootcamp](https://www.alchemy.com/university/courses/ethereum) - Alchemy University
 * [Learn Blockchain](https://www.youtube.com/playlist?list=PLlp912GlUiC1xPnwVmKgIiuMERetMwre9) - Roomyan
+* [Learn Blockchain & Solidity Development Full Course 2024](https://www.youtube.com/playlist?list=PLS5SEs8ZftgUNcUVXtn2KXiE1Ui9B5UrY) - Gregory (Dapp University)
```

Comment:
Don't invent titles! Use the title given by the resource.
for Youtube, use the channel name - "Dapp University", not "Gregory"

Reply:
@eshellman fixed.